iso/scripts/common_iso_generation.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
could you add comments explaining that this script is used either by generating debian and ubuntu iso's scripts